pkg/virt-controller/watch/vmi_test.go

```diff
@@ -324,6 +324,38 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			table.Entry(", ready and in failed state", k8sv1.PodFailed, true),
 			table.Entry(", ready and in pending state", k8sv1.PodPending, true),
 		)
+
+		Context("when pod failed to schedule", func() {
+			It("should set scheduling pod condition on the VirtualMachineInstance", func() {
```

Comment:
Can you add  a test which will remove the condition?

Reply:
Done